ci/build_cpp.sh

```diff
@@ -16,8 +16,28 @@ rapids-logger "Begin cpp build"
 
 sccache --zero-stats
 
-# This calls mambabuild when boa is installed (as is the case in the CI images)
-RAPIDS_PACKAGE_VERSION=$(rapids-generate-version) rapids-conda-retry mambabuild conda/recipes/librmm
+RAPIDS_PACKAGE_VERSION=$(rapids-generate-version)
+export RAPIDS_PACKAGE_VERSION
+
+# rattler-build doesn't have built-in support for GIT_DESCRIBE_NUMBER and
+# GIT_DESCRIBE_HASH so we set them in the environment first
+# shellcheck disable=SC2034
+IFS=- read -r TAG_VERSION GIT_DESCRIBE_NUMBER GIT_DESCRIBE_HASH <<< "$(git describe --tags)"
+unset TAG_VERSION
+export GIT_DESCRIBE_NUMBER
+export GIT_DESCRIBE_HASH
+
+mamba install rattler-build -c conda-forge -y
```

Comment:
Adding this to our CI images with PR: https://github.com/rapidsai/ci-imgs/pull/234